overrides/cf.d.ts

```diff
@@ -121,6 +121,12 @@ interface RequestInitCfProperties {
    * Only available for Enterprise customers.
    */
   cacheKey?: string;
+  /**
+   * Allows for appending additional tags to the Response Headers being returned from the origin
+   * When an asset is uncached, this will be included as a relevant tag for the cached asset
+   * This provides a greater deal of control over things like Flex Purge
```

Comment:
Change `Flex Purge` to the external product name.

Reply:
Done!